.github/workflows/shellcheck.yml

```diff
@@ -0,0 +1,15 @@
+name: ShellCheck
+on:
+  push:
+    paths:
+      - "scripts/**"
+  pull_request:
+    paths:
+      - "scripts/**"
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - run: shellcheck ./scripts/*
```

Comment:
github check fails because of the webhook-cert directory https://github.com/kubewarden/kubewarden-controller/pull/85/checks?check_run_id=3721380603

Reply:
This should be fixed now